onnx/backend/test/loader/__init__.py

```diff
@@ -53,10 +55,15 @@ def load_model_tests(data_dir=os.path.join(DATA_DIR, 'model')):
 
     for test_name in os.listdir(data_dir):
         case_dir = os.path.join(data_dir, test_name)
-        with open(os.path.join(case_dir, 'data.json')) as f:
-            data = json.load(f)
-            url = data['url']
-            model_name = data['model_name']
+        if os.path.exists(os.path.join(case_dir, 'model.pb')):
+            test_name = os.path.basename(case_dir)
+            url = case_dir
```

Comment:
I would prefer to extend ModelTestCase to have one more field `model_dir` (or `data_dir`)